cpp/tests/strings/integers_tests.cu

```diff
@@ -26,48 +26,167 @@
 #include <string>
 #include <vector>
 
+// Using an alias variable for the null elements
+// This will make the code looks cleaner
+constexpr auto NULL_VAL = 0;
+
+using FCW_B   = cudf::test::fixed_width_column_wrapper<bool>;
+using FCW_I16 = cudf::test::fixed_width_column_wrapper<int16_t>;
+using FCW_I32 = cudf::test::fixed_width_column_wrapper<int32_t>;
+using FCW_I64 = cudf::test::fixed_width_column_wrapper<int64_t>;
+using FCW_U32 = cudf::test::fixed_width_column_wrapper<uint32_t>;
+using STR_CW  = cudf::test::strings_column_wrapper;
+using STR_CV  = cudf::strings_column_view;
+using INZ_B   = std::initializer_list<bool>;
+using INZ_I8  = std::initializer_list<int8_t>;
+using INZ_I16 = std::initializer_list<int16_t>;
+using INZ_I32 = std::initializer_list<int32_t>;
+using INZ_U32 = std::initializer_list<uint32_t>;
```

Comment:
Why? This is incredibly unreadable.

Reply:
I renamed them, now they look much more understandable. Reducing from a long name like `cudf::test::fixed_width_column_wrapper<int32_t>`---48 characters, to `COLW_Int32`---10 characters, can save a ton of keystrokes and make the code significantly less clumsy. Especially with the limits of 100 chars per line, long names very frequently break my code lines into more lines. That is so annoying.

Reply:
I disagree. Any decent editor will help with the keystrokes. Can you change it back please?

Reply:
Alright, I reverse back.

Reply:
> I disagree. Any decent editor will help with the keystrokes. Can you change it back please?

You have to type some amount of keys before IntelliSense is triggered though. My typing speed is high thus most of the time I finish typing the entire word before IntelliSense kicks in.